bookwyrm/views/follow.py

```diff
@@ -84,3 +94,68 @@ def delete_follow_request(request):
 
     follow_request.delete()
     return redirect(f"/user/{request.user.localname}")
+
+
+def ostatus_follow_request(request):
+    """prepare an outgoing remote follow request"""
+    uri = urllib.parse.unquote(request.GET.get("acct"))
+    username_parts = re.search(
+        r"(?:^http(?:s?):\/\/)([\w\-\.]*)(?:.)*(?:(?:\/)([\w]*))", uri
+    )
+    account = f"{username_parts[2]}@{username_parts[1]}"
+    user = handle_remote_webfinger(account)
+    error = None
+
+    if user is None or user == "":
+        error = "ostatus_subscribe"
+
+    # don't do these checks for AnonymousUser before they sign in
+    if request.user.id:
```

Comment:
You can check `request.user.is_authenticated:` -- it'll be functionally the same, but it's a little clearer what's being checked.